frontend/www/js/omegaup/components/problem/Tags.vue

```diff
@@ -19,17 +19,21 @@
       <table class="table table-striped">
         <thead>
           <tr>
-            <th class="text-center" scope="col">
+            <th class="text-center" width="50%" scope="col">
               {{ T.contestEditTagName }}
             </th>
-            <th class="text-center" scope="col">
+            <th
+              style="text-align: left; width: 50%; padding-left: 15%"
```

Comment:
Same comment as previous one. 

And one more thing: We discourage the use of `style` attribute because of 
 possible xss attacks. Instead can you use `bootstrap-4` classes:
```suggestion
              class="text-left w-50 pl-1"
```
?
It applies in Line 90 too.